options.go

```diff
@@ -334,3 +343,69 @@ func WithWaitStrategyAndDeadline(deadline time.Duration, strategies ...wait.Stra
 		return nil
 	}
 }
+
+// WithEntrypoint appends the entrypoint to the entrypoint for a container
+func WithEntrypoint(entrypoint ...string) CustomizeRequestOption {
+	return func(req *GenericContainerRequest) error {
+		req.Entrypoint = append(req.Entrypoint, entrypoint...)
+		return nil
+	}
+}
+
+// WithExposedPorts appends the ports to the exposed ports for a container
+func WithExposedPorts(ports ...string) CustomizeRequestOption {
```

Comment:
question: do we want to pass strings, or nat.Port? :thinking:

Reply:
my initial reaction as string, but nat.Port might have it's benefits. The key for me is usability, can I do:
```go
testcontainers.WithExposedPorts("8080")
```

Reply:
Yes, because nat.Port is declared as a new string type:

```go
// Port is a string containing port number and protocol in the format "80/tcp"
type Port string
```

Reply:
Yer it works for literals but not for variables unless defined correctly, is that an issue?
Example: https://go.dev/play/p/nfjRMSr94-G

Reply:
let use string then